buddyboss/buddyboss-platform
# activity embed block when link preview used

### All Submissions:






### Changes proposed in this Pull Request:



Closes #341  .

### How to test the changes in this Pull Request:

1. Go to activity post form
2. Paste https://buddyboss.com url and save with link preview in post.
3. It should show only link preview without any embeds in the activity.

### Proof Screenshots or Video

![image](https://user-images.githubusercontent.com/4090336/71823922-64209d80-30be-11ea-8970-b125ed9e8b48.png)

### Other information:




### Changelog entry

> Activity > Double link preview of URLs fixed
